www/nginx/src/opnsense/mvc/app/controllers/OPNsense/Nginx/forms/location.xml

```diff
@@ -71,13 +71,15 @@
     <id>location.index</id>
     <label>Index File</label>
     <type>select_multiple</type>
-    <help>Enter a list of file extensions, which are served instead of a directory. It is common to use index.html or index.php here.</help>
+    <help>Enter a list of file extensions, which are served instead of a directory. It is common to use index.html or
```

Comment:
would keep these on single lines as the whitespace noise is going into the translations/changing them